### ISSUE TITLE
TileDb.Run() never exits

### ISSUE THREAD

**Issue by nkovacs** [fawick/go-mapnik#10]
[This line](https://github.com/fawick/go-mapnik/blob/master/maptiles/mbtilesgenerator.go#L123) is unreachable.

**Comment by fawick** [comment]
You are right again. That was work-in-progress which I never addressed again. Thanks for reviewing and reporting that. You already have a fix for that, right? I'd be happy to take in a PR.
